syft/generic/object.py

```diff
@@ -157,6 +155,7 @@ def handle_func_command(cls, command):
 
         # TODO: I can't manage the import issue, can you?
         # Replace all LoggingTensor with their child attribute
+        # TODO[jvmancuso]: get rid of these torch references when extending hook_args
```

Comment:
Is this done?

Reply:
nope, will be in a soon-to-be PR

Reply:
will make an issue for it though

Reply:
https://github.com/OpenMined/PySyft/issues/2530